src/vmod_var.c

```diff
@@ -172,6 +177,10 @@ vmod_set_string(const struct vrt_ctx *ctx, struct vmod_priv *priv,
 	if (name == NULL)
 		return;
 	v = vh_get_var_alloc(get_vh(priv), name, ctx);
+	if (v == NULL)
```

Comment:
code style: no parenthesis when there's only one line in an `if`/`for`/`while`

Reply:
Done!